lib/player/player.go

```diff
@@ -81,7 +81,12 @@ type Player struct {
 	translator sessionPrintTranslator
 }
 
-const normalPlayback = math.MinInt64
+const (
+	normalPlayback = math.MinInt64
+	// MaxIdleTimeMilliseconds defines the max idle time when skipping idle
+	// periods on the recording.
+	MaxIdleTimeMilliseconds = 500.0
```

Comment:
Can we convert this and the calculations into `time.Duration` ones instead? `Player.speed` being an `atomic.Value` is also pretty awful.

Reply:
I've converted the timing values into `time.Duration`.

About the `speed` attribute, it needs lock protection, given that users might change the playback speed at any time during the session replay (through the `SetSpeed` function). I'm unsure if your comment relates to the necessity of the atomic value or the usage of `atomic.Value` specific. I will go ahead and merge this PR so we can have the fix released, and I can open a follow-up on this refactor (if we require it).

Reply:
The speed value being atomic makes sense, using an `atomic.Value` is really awful, forcing allocations and type assertions every time we interact with it.

If we ever refactor this we should just use an `atomic.Uint64` and `math.Float64bits`/`Float64frombits` instead - it's not worth doing a change specifically for this, however.